src/hidpp20.h

```diff
@@ -699,7 +699,8 @@ struct hidpp20_profile {
 	uint16_t dpi[HIDPP20_DPI_COUNT];
 	union hidpp20_button_binding buttons[32];
 	union hidpp20_macro_data *macros[32];
-	struct hidpp20_led leds[HIDPP20_LED_COUNT];
+	struct hidpp20_led leds_8070[HIDPP20_LED_COUNT];
+	struct hidpp20_led leds_8071[HIDPP20_LED_COUNT];
```

Comment:
i don't quite understand why we need to duplicate this here - are there mice that have both features? if not, then we don't need the same structs duplicated, we can switch at runtime where necessary.

Reply:
This struct maps directly to the memory we read from the mouse. We could instead have `struct hidpp20_led leds[HIDPP20_LED_COUNT * 2];` and then have a define for the index, which would be `0` for 0x8070 and `2` for 0x8071, but I think this way is cleaner.

Reply:
I misread the documentation. It seems this 2 new leds describe the effects triggered when the device goes to power save mode.